### PR TITLE
Feature: Add new Personalised daily flow assets + components

### PR DESCRIPTION
**Pull request by imanmzoe**
# Description

- [x] Add carousel
- [x] Add assets 

[comment]: <> (Description of the feature or fix)
[comment]: <> (Include link to Figma designs, if relevant)
[comment]: <> (Include link to Notion ticket, if relevant)

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [ ] iOS - Emulator
- [ ] iOS - Physical device

## Out of scope and potential follow-up

[comment]: <> (Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?)

## Screenshots

[comment]: <> (Please attach screenshots showing the change, if relevant)

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if relevant)
- [ ] Added new screen(s) to `src/features/screens.ts` and new Redux slice(s) to `mocks/mockedInitialState.ts` (if relevant)

## Checklist (for reviewers)

- [ ] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Checklist (for merger)

- [ ] Make sure the right merge strategy is chosen, commit merge into master and squash merge into develop.

### CODE REVIEW

File: assets/v2/utils/dimensions.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  if (config.width && config.height) {
+    width = config.width;
+    height = config.height;
+  } else if (config.width) {
+    width = config.width;
+    height = defaultHeight * (config.width / defaultWidth);
+  } else if (config.height) {
+    width = defaultWidth * (config.height / defaultHeight);
+    height = config.height;
+  }
```
Comment (vanhoutk) [inline assets/v2/utils/dimensions.ts:22]: Can shorten this to

```suggestion
  let width = config.width || defaultWidth * (config.height / defaultHeight);
  let height = config.height || defaultHeight * (config.width / defaultWidth);
```

File: assets/v2/svg/diseases/icon-reproductive-health.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { withDimensions } from '@assets/v2/utils/dimensions';
+import { colors } from '@tracer/branding/colors';
+import React from 'react';
+import { Path, Svg } from 'react-native-svg';
+
+import { IconDiseaseProps } from './types';
+
+const defaultWidth = 25;
+const defaultHeight = 41;
```
Comment (vanhoutk) [inline assets/v2/svg/diseases/icon-reproductive-health.tsx:9]: In future it'd be handy if our icons/svgs had a consistent structure to them (re: sizing). Something to discuss with Maria.

File: assets/v2/utils/dimensions.ts (hunk 1 above)
Comment (imanmzoe) [reply assets/v2/utils/dimensions.ts]: nice - was just copy-pasted from what was there before

**Review by vanhoutk — APPROVED** [review]
